python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/monitor-sessions.j2

```diff
@@ -54,10 +54,18 @@
 {%             endif %}
 {%         endif %}
 {%     endfor %}
+{%     if monitor_sessions_default_encapsulation_gre is arista.avd.defined %}
+
+### Monitor Sessions Default
+
+| Settings | Values |
```

Comment:
```suggestion
| Settings | Values |
| -------- | ------- |
```

Reply:
I think there is an extra "-" below values in your suggestion here

Reply:
Corrected.

Reply:
Fixed!